templates/default/partials/version/system-requirements.html.twig

```diff
@@ -1,5 +1,6 @@
 {% frame with { id: 'system-requirements', color: 'lighter', title: 'System Requirements' } %}
-    <p>For more information and installation instructions read the <a href="https://docs.typo3.org/m/typo3/tutorial-getting-started/master/en-us/SystemRequirements/Index.html" rel="noopener,noreferrer">Installation guide</a>.</p>
+    <p>For more information and installation instructions read the
+        <a href="https://docs.typo3.org/permalink/t3coreapi:system-requirements" rel="noopener,noreferrer">Installation guide</a>.</p>
```

Comment:
`noreferrer` already implies `noopener`: https://developer.mozilla.org/en-US/docs/Web/HTML/Reference/Attributes/rel/noreferrer

However, `noopener` should be sufficient.